### PR TITLE
fix clippy

### PR DESCRIPTION
**Pull request by PSeitz**
fix clippy

### CODE REVIEW

**Comment by fulmicoton** [comment]
if it is not performance critical i'd rather have boxing T_T

**Comment by PSeitz** [comment]
In this case there are a lot of tests that would need to be adjusted

**Comment by fulmicoton** [comment]
@PSeitz let's adjust the tests

File: src/aggregation/agg_tests.rs [hunk 1]
```diff
@@ -590,7 +590,7 @@ fn test_aggregation_invalid_requests() -> crate::Result<()> {
     Ok(())
 }
 
-#[cfg(all(test, feature = "unstable"))]
+//#[cfg(all(test, feature = "unstable"))]
```
Comment (fulmicoton) [inline src/aggregation/agg_tests.rs]: this is a mistake i believe?

**Review by fulmicoton — APPROVED** [review]
approved to make the exchange smooth, but have a look at the comment in agg_test.
I think you commented out some feature fence by mistake.

**Pull request merged**